scripts/autostart.lic

```diff
@@ -76,6 +79,27 @@ def lich_no_update
 	end
 end
 
+if XMLData.game =~ /^DR/
+  begin
+    did_dependency_install = Lich.db.get_first_value("SELECT value FROM lich_settings WHERE name='did_dependency_install';")
+  rescue SQLite3::BusyException
+    sleep 0.1
+    retry
+  end
+  if did_dependency_install.nil?
+    unless File.exist?("#{SCRIPT_DIR}/bootstrap.lic") && File.exist?("#{SCRIPT_DIR}/drinfomon.lic")
```

Comment:
```suggestion
    unless File.exist?("#{SCRIPT_DIR}/dependency.lic")
    Script.run('repository', 'download dependency')
    sleep 1
    end
    unless File.exist?("#{SCRIPT_DIR}/bootstrap.lic") && File.exist?("#{SCRIPT_DIR}/drinfomon.lic")
```

Reply:
Splitting up the download action to account for folks downloading dependency elsewhere (or in my case, trying to test dependency out of band)